rtv/packages/praw/objects.py

```diff
@@ -1688,7 +1688,7 @@ def __init__(self, reddit_session, author=None, name=None,
                  json_dict=None, fetch=False, **kwargs):
         """Construct an instance of the Multireddit object."""
         author = six.text_type(author) if author \
-            else json_dict['path'].split('/')[-3]
+            else json_dict['path'].split('/')[-4]
```

Comment:
Yes it's a bugfix:
`author = "/user/multi-mod/m/android".split('/')[-4] = "multi-mod"`